Articles/Defining and Using Observables.md

```diff
@@ -0,0 +1,341 @@
+
+As with most things in OCSF, these definitions can be in the base of the core schema, one of the core schema extensions, or any other private extension (those extensions outside of the core schema).
+
+As a historical note, definition types 1 and 3 have been in use since schema version 1.0, and the rest became available for use since 1.2.
+
+### Definition Example: Observable by Dictionary Type
+Defining an observable by dictionary type is, naturally, done in a metaschema `dictionary.json` file. The definition is done by adding an `observable` field to a type definition.
+
+This, along with defining observable objects (which are also a kind of type) are the broadest ways to define observables. All instances attributes of this type (regardless of attribute name) become observables.
+
+Example in a `dictionary.json` file:
+```jsonc
+{
+  "name": "dictionary",
+  // ... other dictionary fields (caption, description, attributes)
+  "types": {
+    // ... other "types" fields (caption, description)
+    "attributes": {
+      // ... other types
+      "email_t": {
+        
+        // This is the observable definition for the email_t type, a subtype of string_t
+        "observable": 5,
+
+        "caption": "Email Address",
+        "description": "Email address. For example: <code>john_doe@example.com</code>.",
+        "regex": "^[a-zA-Z0-9!#$%&'*+-/=?^_`{|}~.]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$",
+        "type": "string_t",
+        "type_name": "String"
+      },
+      // ...
+    }
+  }
+}
+```
+
+### Definition Example: Observable by Dictionary Attribute
+Defining an observable by dictionary attribute is also done in a metaschema `dictionary.json` file. The definition is done by adding an `observable` field to an attribute definition. 
+
+Definitions done this way limit the creation of observables instances of this specific attribute, regardless of where it is used.
+
+Example in a `dictionary.json` file:
+```jsonc
+{
+  "name": "dictionary",
+  // ... other dictionary fields (caption, description, types)
+  "attributes": {
+    // ... other attributes
+    "cmd_line": {
+      "caption": "Command Line",
+      // ... other attribute fields
+
+      // This is the observable definition for the cmd_line attribute
+      "observable": 13
+    },
+    // ...
+  },
+  // ...
+}
+```
+
+### Definition Example: Observable by Object
+Defining an observable by object is in a metaschema object definition file. The definition is done by adding an `observable` field directly in the object definition object. This also works for object definitions that extend another, including the special patch case. For both regular and patch extends cases, the observable definition adds or replaces any existing object-level observable definition.
```

Comment:
Grammar: "Defining an observable by object is in a metaschema object definition file." Should be '...by object is done in a metaschema object definition file.'